ci/run-k8s-e2e-tests.sh

```diff
@@ -43,8 +43,8 @@ THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 KUBE_CONFORMANCE_IMAGE_VERSION_OPTION=""
 IMAGE_PULL_POLICY="Always"
 CONFORMANCE_IMAGE_CONFIG_PATH="${THIS_DIR}/conformance-image-config.yaml"
-SONOBUOY_IMAGE="projects.registry.vmware.com/sonobuoy/sonobuoy:v0.56.16"
-SYSTEMD_LOGS_IMAGE="projects.registry.vmware.com/sonobuoy/systemd-logs:v0.4"
+SONOBUOY_IMAGE="sonobuoy/sonobuoy:v0.56.16"
```

Comment:
Cloud you help upload sonobuoy/sonobuoy:v0.56.16 and sonobuoy/systemd-logs:v0.4 to antrea dockerhub? I suspect these images still have rate limit issues in sonobuoy dockerhub. cc @tnqn @antoninbas

Reply:
Done